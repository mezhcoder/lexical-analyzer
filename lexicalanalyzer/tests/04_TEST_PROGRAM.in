program foo;
var x, y: integer;
begin end.