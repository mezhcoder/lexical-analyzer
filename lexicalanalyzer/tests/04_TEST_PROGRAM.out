1	1	KEYWORD	program	program
1	9	ID	foo	foo
1	12	;
2	1	KEYWORD	var	var
2	5	ID	x	x
2	6	,
2	8	ID	y	y
2	9	:
2	11	ID	integer	integer
2	18	;
3	1	KEYWORD	begin	begin
3	7	KEYWORD	end	end
3	10	.	.	.